protocol-designer/src/components/organisms/AssignLiquidsModal/LiquidToolbox.tsx

```diff
@@ -273,12 +278,35 @@ function LiquidToolbox({
       : false
   const hasSelection = selectedWells.length > 0
   const canEdit = !showLiquidLayoutOverlay && (hasLiquids || hasSelection)
+  const { labware, modules } = useSelector(getInitialDeckSetup)
+  const stackerModuleIds = Object.entries(modules).reduce<string[]>(
+    (result, [key, value]) => {
+      return value.type === FLEX_STACKER_MODULE_TYPE
+        ? result.concat(key)
+        : result
+    },
+    []
+  )
```

Comment:
this works but it would be nice to make things a bit more descriptive. The result is actually the `stackerIds` that = the stacker, the key is the `moduleId`, the value is the `moduleOnDeck`

```suggestion
  const stackerModuleIds = Object.entries(modules).reduce<string[]>(
    (stackerIds, [moduleId, moduleOnDeck]) => {
      return moduleOnDeck.type === FLEX_STACKER_MODULE_TYPE
        ? stackerIds.concat(moduleId)
        : stackerIds
    },
    []
  )
```